docs/changehistory/NextVersion.md

```diff
@@ -1599,178 +1334,493 @@ Similarly, if your existing code passes an object instead of an array as the que
+| `DiagnosticCodes`                        | `DiagnosticCodes` in @itwin/ecschema-editing               |
+| `Diagnostics`                            | `Diagnostics` in @itwin/ecschema-editing                   |
+| `IDiagnosticReporter`                    | `IDiagnosticReporter` in @itwin/ecschema-editing           |
+| `SuppressionDiagnosticReporter`          | `SuppressionDiagnosticReporter` in @itwin/ecschema-editing |
+| `FormatDiagnosticReporter`               | `FormatDiagnosticReporter` in @itwin/ecschema-editing      |
+| `LoggingDiagnosticReporter`              | `LoggingDiagnosticReporter` in @itwin/ecschema-editing     |
+| `IRuleSet`                               | `IRuleSet` in @itwin/ecschema-editing                      |
+| `ECRuleSet`                              | `ECRuleSet` in @itwin/ecschema-editing                     |
+| `ISuppressionRule`                       | `ISuppressionRule` in @itwin/ecschema-editing              |
+| `BaseSuppressionRule`                    | `BaseSuppressionRule` in @itwin/ecschema-editing           |
+| `IRuleSuppressionMap`                    | `IRuleSuppressionMap` in @itwin/ecschema-editing           |
+| `BaseRuleSuppressionMap`                 | `BaseRuleSuppressionMap` in @itwin/ecschema-editing        |
+| `IRuleSuppressionSet`                    | `IRuleSuppressionSet` in @itwin/ecschema-editing           |
+| `SchemaCompareCodes`                     | `SchemaCompareCodes` in @itwin/ecschema-editing            |
+| `SchemaCompareDiagnostics`               | `SchemaCompareDiagnostics` in @itwin/ecschema-editing      |
+| `SchemaValidater`                        | `SchemaValidater` in @itwin/ecschema-editing               |
+| `SchemaValidationVisitor`                | `SchemaValidationVisitor` in @itwin/ecschema-editing       |
+| `RelationshipConstraint.deserialize`     | `RelationshipConstraint.fromJSON`                            |
+| `RelationshipConstraint.deserializeSync` | `RelationshipConstraint.fromJSONSync`                        |
+| `RelationshipConstraint.toJson`          | `RelationshipConstraint.toJSON`                              |
 
-On the frontend the [GeoConverter]($frontend) class has been modified to accept either a string containing the datum or a [GeographicCRSProps]($common) of a similar format retaining cache capability as before for either format.
+### @bentley/itwin-client
 
-**NOTE**: The [IModelCoordinatesRequestProps.source]($common) and the [GeoCoordinatesRequestProps.target]($common) were renamed from previous version that used the sourceDatum and targetDatum properties.
+| Removed                            | Replacement                    |
+| ---------------------------------- | ------------------------------ |
+| `UserInfo`                         | *eliminated*                   |
+| `AuthorizationClient.isAuthorized` | *eliminated*                   |
+
+### @bentley/appui-react
```

Comment:
Looks like content of this topic should be merged with content of `@itwin/appui-react`

Reply:
Consolidated.